conf/logback.xml

```diff
@@ -28,7 +28,7 @@
 
     <logger name="com.google.inject" level="INFO"/>
 
-    <logger name="uk.gov" level="INFO"/>
+    <logger name="uk.gov" level="DEBUG"/>
```

Comment:
Is this required?
This effects change in log scope for all services in MDTP we depend on. I would leave this at INFO

If it's the new debug scope logs we have introduced for tax years in this service, the existing `application` logger already caters for `DEBUG` scope.